public/index.html

```diff
@@ -0,0 +1,15 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Buy yourself a Lambo!</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.0/css/all.min.css"
+    />
+    <script defer src="./bundle.js"></script>
```

Comment:
We talked about this in voices but just wanted to log it in here too - be careful about using relative paths in your index.html - it might cause some weird bugs